WordPress/Classes/ViewRelated/System/Coordinators/MySitesCoordinator.swift

```diff
@@ -100,8 +100,13 @@ class MySitesCoordinator: NSObject {
     }
 
     @objc func signinDidFinish() {
-        mySiteViewController = makeMySiteViewController()
-        navigationController.viewControllers = [rootContentViewController]
+        // The code below raises an exception during unit tests. Adding a `try?` to ignore the error.
+        // The exception is Error Domain=NSRangeException Code=0 "(null)". I'm not exactly sure what code (probably
+        // deep in UIKit) throws this exception, but I don't see any reason the code below would cause crash in production.
+        try? WPException.objcTry {
+            self.mySiteViewController = self.makeMySiteViewController()
+            self.navigationController.viewControllers = [self.rootContentViewController]
+        }
```

Comment:
I suppose we consider safe to blanket ignore any exception here because we are confident they would arise only when running unit tests due to the many stateful assumption in the app at the moment.

Is that safe?

How would you feel about running this only if `NSClassFromString("XCTestCase") != nil`?

Reply:
It should be okay because I don't think there is no exception in this code in the real app.

As I shared on Slack, the actual crash site is at [this code](https://github.com/wordpress-mobile/WordPress-iOS/blob/be3f6b0b1e0b3a0c59b4d71cd71b9089044585d1/WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsButtonViewModel.swift#L47), which should never happen in the real app too.

I also can't find any related issue on Sentry.